test/docker/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM ruby:2.2
+FROM ruby:2
 
-ENV DOCKER_VERSION 1.10.1-0~jessie
+ENV DOCKER_VERSION 1.13.1-0~debian-jessie
 
 RUN apt-get update && \
     apt-get install -y make apt-transport-https ca-certificates && \
-    apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 \
-      --recv-keys 58118E89F3A912897C070ADBF76221572C52609D && \
+    (apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D \
```

Comment:
get the idea from this PR, https://github.com/SonarSource/docker-sonarqube/pull/205/files#diff-fda375794afbb145346f7b8f025bd00cR27

But the GPG issue is pretty common thing.